packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,256 @@ def _clean_html(self, html: str) -> str:
-                    # Check if there are more pages
-                    total_size = response.get("totalSize", response.get("size", 0))
-                    if start + limit >= total_size:
-                        logger.debug(
-                            f"Reached end of results: {start + limit} >= {total_size}"
-                        )
-                        break
+                # Check if there are more pages
+                total_size = response.get("totalSize", response.get("size", 0))
+                if start + limit >= total_size:
+                    logger.debug(
+                        f"Reached end of results: {start + limit} >= {total_size}"
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)